demos/serverless/src/handlers.js

```diff
@@ -32,6 +32,13 @@ const getMeeting = async(meetingTitle) => {
     return null;
   }
   const meetingData = JSON.parse(result.Item.Data.S);
+  try {
```

Comment:
So from what I understand, chime.getMeeting with an existing ID of a meeting that already ended will return an error that we will catch and return null instead. Is that correct?
We have a integ test to verify this behavior where you cannot rejoin an ended meeting. Will this break that test?